test/test_fx_experimental.py

```diff
@@ -563,8 +566,18 @@ def test_conv_bn_fusion_mixed_dtype(self):
         class M(torch.nn.Module):
             def __init__(self) -> None:
                 super().__init__()
-                self.conv = torch.nn.Conv2d(3, 16, kernel_size=3, stride=1, padding=1, bias=False, dtype=torch.bfloat16)
-                self.bn = torch.nn.BatchNorm2d(16, eps=0.001, momentum=0.1, affine=True, track_running_stats=True)
+                self.conv = torch.nn.Conv2d(
+                    3,
+                    16,
+                    kernel_size=3,
+                    stride=1,
+                    padding=1,
+                    bias=False,
+                    dtype=torch.bfloat16,
+                )
+                self.bn = torch.nn.BatchNorm2d(
+                    16, eps=0.001, momentum=0.1, affine=True, track_running_stats=True
+                )
```

Comment:
bah why you reformat unrelated code

Reply:
Hmm not sure what happened here - I think some linter was automatically applied.

Reply:
I have found the usual explanation  is you pyfmt version doesn't match the repo. Lintrunner init will fix it